src/utilities/FormHelpers.js

```diff
@@ -56,9 +54,9 @@ export const URLRouting = (categories, categoryId) =>{
 		}
 		else
 		{
-			const getSelectedCategory = _.filter(categories, { id: categoryId });
-			nameCat = (getSelectedCategory.length > 0) ? getSelectedCategory[0].name : '' ;
-			idCat =  (getSelectedCategory.length > 0) ? getSelectedCategory[0].id : '' ;
+			const getSelectedCategory = categories.find(items => (items.id.toString() === categoryId));
```

Comment:
this can be simplified using a category helper.

https://github.com/baltimorecounty/react-baltcogo/blob/580cdb37410b63d75a0a72045e3337fa110bd3e2/src/utilities/CategoryHelpers.js#L1